server/src/weaverbird/backends/pypika_translator/translators/base.py

```diff
@@ -1390,19 +1476,17 @@ def text(
         columns: list[str],
         step: "TextStep",
     ) -> StepContext:
-        from pypika.terms import ValueWrapper
-
         # Since we're using WITH...AS syntax, we add an explicit cast here to provide type
         # context to the engine. Without that, we might encounter "failed to find conversion
         # function from "unknown" to text" errors
         value = step.text
 
         if isinstance(value, datetime):
-            # ValueWrapper(value) would produce an iso8601 string which
+            # ValueWrapperCustom(value) would produce an iso8601 string which
             # is not properly handled by some backends
-            value_wrapped = ValueWrapper(value.strftime("%Y-%m-%d %H:%M:%S"))
+            value_wrapped = ValueWrapperCustom(value.strftime("%Y-%m-%d %H:%M:%S"))
```

Comment:
This behaviour is only for GBQ. Changing the behaviour here breaks all other translators (see the added fixture)